api/api.py

```diff
@@ -334,6 +348,54 @@ def historical(self, request, pk=None):
     def teste(self, request, pk=None):
         return Response("ola teste")
 
+    @action(detail=False)
+    def expenses_revenue_year_by_country(self, request, pk=None):
+        """
+            This is an API for a country's budget list.
+        """
+        params = ExpensesRevenueYearFilterSerializer(data=request.GET)
+        params.is_valid(raise_exception=True)
+        country = params.validated_data.get("country")
+        # country = Country.objects.get(country=country)
+        budget_summaries = BudgetSummary.objects.order_by("budget__year").filter(budget__country=country)
+        list_dict = []
+        for budget_summary in budget_summaries:
+            if not (budget_summary.expense_functional_budget is None):
```

Comment:
Logic can be improved with python: `if not (budget_summary.expense_functional_budget is None):` => `if budget_summary.expense_functional_budget:`